resources/deb/triggers.erb

```diff
@@ -0,0 +1,17 @@
+<%- unless get_activate_triggers.empty? -%>
+<%- get_activate_triggers.each do |trigger| -%>
+activate <%= trigger %>
+<% end -%>
+<%- end -%>
+
+<%- unless get_interest_triggers("install").empty? -%>
+<%- get_interest_triggers("install").each do |trigger| -%>
+interest <%= trigger.interest_name %>
+<% end -%>
+<%- end -%>
+
+<%- unless get_interest_triggers("upgrade").empty? -%>
+<%-	get_interest_triggers("upgrade").each do |trigger| -%>
```

Comment:
If `#get_interest_triggers` returns an empty Iterable object (like an empty array), then you can lose the `get_interest_triggers("upgrade").empty?` test because Ruby (and therefore ERB) will happily skip over doing anything inside the loop if the Iterable object is empty.

Reply:
@demophoon ^^^